src/libs/ReportUtils.ts

```diff
@@ -3153,7 +3153,7 @@ function getReportName(report: OnyxEntry<Report>, policy: OnyxEntry<Policy> = nu
             if (isArchivedRoom(report)) {
                 formattedName += ` (${Localize.translateLocal('common.archived')})`;
             }
-            return formattedName;
+            return formattedName.replace(/(\r\n|\n|\r)/gm, ' ');
```

Comment:
https://github.com/Expensify/App/blob/e8ae3c5acedf0e6788dc574c7b6f3043ca37092a/src/libs/ReportUtils.ts#L3155

Should we apply the same fix in here?

Reply:
@nkdengineer Could we use `formatReportLastMessageText` function instead of replacing manually?

Reply:
> Should we apply the same fix in here?

I think we can apply the same for this to prevent the same bug in the feature.

> Could we use formatReportLastMessageText function instead of replacing manually?

Yes we can use this function.